test/TestAppCX/CornerRadiusTestPage.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Page
+    x:Class="TestAppCX.CornerRadiusTestPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TestAppCX"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel HorizontalAlignment="Left">
+        <Slider Width="200" Margin="20"/>
+        <ProgressBar Minimum="0" Maximum="100" Value="50" Width="200" Margin="20"/>
+        <CheckBox Content="CheckBox" Margin="20"/>
+        <controls:ColorPicker IsAlphaEnabled="True" Margin="20"/>
+    </StackPanel>
```

Comment:
Can we add this test to the nuget CX test app and actually make sure there's a backstop for this problem on the nuget side?

Reply:
Added